service-catalog/generic-service.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    velocity.tech.v1/id: generic-service-var-service-id
+  name: generic-service-var-service-id
+spec:
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      name: generic-service-var-service-id
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        name: generic-service-var-service-id
+    spec:
+      containers:
+        - image: busybox:1.28
+          command:
+            - sleep
+            - "3600"
+          ports:
+            - name: http
+              containerPort: 3000
+              protocol: TCP
+          imagePullPolicy: IfNotPresent
+          name: generic-service
+      restartPolicy: Always
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: generic-service-var-service-id
+spec:
+  ports:
+    - name: tcp
+      port: 3005
+      protocol: TCP
+      targetPort: 3000
+  selector:
+    name: generic-service-var-service-id
+  type: ClusterIP
```

Comment:
lets delete the service - it doesn't do anything

Reply:
without Service, I can't run development for  generic-service

Reply:
it's ok, the editor will add this service once needed.
for now, it is not being used, and for starters we need a clean slate for a service.